int8 type
int8 gripType # Depends on NOSTOP, STOPHAND, STOPFINGER, CLOSINGGRIP
float64 ratio
float64 velocity
int8 PARALLEL=0
int8 CENTRICAL=1
int8 CYLINDRICAL=2
int8 SPHERICAL=3
int8 HOME=4
int8 NOSTOP=0
int8 STOPHAND=1
int8 STOPFINGER=2
int8 CLOSINGGRIP=3
---
bool result
